Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/audit/PatientDiscoveryAuditTransformConstants.java

```diff
@@ -27,10 +27,11 @@
 package gov.hhs.fha.nhinc.patientdiscovery.audit;
 
 /**
+ * Constants specific to the Patient Discovery AuditLogger implementation.
  *
  * @author achidamb
  */
-public class PatientDiscoveryTransformConstants {
+public class PatientDiscoveryAuditTransformConstants {
```

Comment:
should be consistent if we're renaming from Transform to Transforms.  Change to "PatientDiscoveryAuditTransformsConstants"